src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -2343,11 +2343,11 @@ class CgroupUtils(object):
                                      self.cfg['cgroup_prefix'] + '.slice')
         try:
             with open(slicefile, 'w') as desc:
-                desc.write('[Unit]\n' +
-                           'Description=%s\n' % description +
-                           '[Slice]\n' +
-                           'Delegate=yes\n' +
-                           'TasksMax=infinity\n')
+                desc.write(('[Unit]\n'
+                            'Description=%s\n'
+                            '[Slice]\n'
+                            'Delegate=yes\n'
+                            'TasksMax=infinity\n') % description)
```

Comment:
No need for the extra set of ().  You're in a set of () by being a function call. (and for other similar strings in function calls)

Reply:
Done.